### ISSUE TITLE
Sidebar open parameter crash Shinylive when passing a list

### ISSUE THREAD

**Issue by cedricbouffard** [rstudio/bslib#1081]
When passing a list to the open parameter in a sidebar, shinylive will display this error:
![image](https://github.com/rstudio/bslib/assets/80600509/b41459d6-72ea-48fe-9cff-5d927b7eba47)

I tried with a named character vector, same problem.

Here's a sample code:

``` r
library(shiny)
library(bslib)

ui <- page_sidebar(

  sidebar = sidebar(
    open = list(desktop = "always", mobile = "open")
  ),
  mainPanel(
    "Test"
  )
)

server <- function(input, output, session) {
 
  }
  


shinyApp(ui, server)

```

**Comment by gadenbuie** [comment]
I wonder if there's an encoding issue? I just copied your example [into shinylive](https://shinylive.io/r/editor/#code=NobwRAdghgtgpmAXGKAHVA6ASmANGAYwHsIAXOMpMAGwEsAjAJykYE8AKAZwAtaJWAlAB0IdJiw71OY4RBEBXWgAIAPAFolqKAHM4AfU60AJnHot2IpUsMmzjJQF5rx0+ctWlRVBUdK6nUnYTTgBrUi9fIRRqAHcoVk4o3CUYInpaajhIsC8KKIFcdysogGUXOyj3AvcogBU4AMqIWRFOOEYAN3bVDQAzeQgCUloSdj5UeVJkokmJqesGwxIBJRAREQBfdYgePlYAQXR2RWS2zvaBMA2AXSA) and it worked as expected. (I edited it lightly in that link but the direct copy works as well.)

I noticed that the error message appears to say

```
Error: `open` must be one of "desktop", "open", "closed", or "always", not "always".
```

which seems to indicate there's a difference between `"always"` and `"always"`?
